src/EPR.Calculator.API.UnitTests/Builder/Summary/OnePlus2A2B2C/OnePlus2A2B2CProducerTests.cs

```diff
@@ -0,0 +1,481 @@
+﻿namespace EPR.Calculator.API.UnitTests.Builder.Summary.OnePlus2A2B2C
+{
+    using EPR.Calculator.API.Builder.Summary.OnePlus2A2B2C;
+    using EPR.Calculator.API.Data;
+    using EPR.Calculator.API.Data.DataModels;
+    using EPR.Calculator.API.Models;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Diagnostics;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    [TestClass]
+    public class OnePlus2A2B2CProducerTests
+    {
+        private ApplicationDBContext _dbContext;
```

Comment:
These fields should be marked as required, otherwise SonarQube will complain.

Reply:
@tim-powell-eviden I have done the fixes